styleguide/source/_patterns/01-molecules/06-components/20-testimonial/testimonial.md

```diff
@@ -0,0 +1,7 @@
+---
+el: ".testimonial"
+title: "Testimonial 50/50"
+---
+A Testimonial 50/50 consists of an Image, a Quote (long text), a Speaker (text), and a Speaker Title (text). There are two variants, a left and a right variant.
```

Comment:
"A Testimonial 50/50 consists of two evenly divided columns. One column contains an Image. The other contains a Quote (long text), a Speaker (text), and a Speaker Title (text). 

There are two variants, a left and a right variant. In this variant, the image displays in the right column. There is also a Testimonial 50/50 Left pattern, which places the image in the left column instead."

@rpama thoughts? I committed this already, but I'm happy to edit.